src/pages/jonghyuk/Main/Newcomment.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Newcomment = props => {
+  return <p>{props.comment}</p>;
+};
```

Comment:
prop은 분해할당 해주세요!

만일 prop으로 한가지만 받는다면 이렇게 작성하는건 어떨까요?

```suggestion
const Newcomment = ({ comment }) => { // prop으로 받는 곳에서 바로 분해
  return <p>{comment}</p>;
};
```